docker/env.list

```diff
@@ -29,3 +29,5 @@ KOLIBRI_RUN_MODE="docker"
 # DOCKERMNT_PEX_PATH=
 # Specify path to a .pex file in the /docker/mnt volume, e.g.
 # DOCKERMNT_PEX_PATH=/docker/mnt/kolibri-0.11.0a1.dev_git-37-gb15b923.pex
+
+BACKGROUND_IMAGE
```

Comment:
maybe we should rename this to something namespaced and build-specific like `KOLIBRI_BUILD_BG_IMAGE` .. somehow distinguish it from kolibri options?